### PR TITLE
add button toggle theme

### LINKED ISSUE #11: Add Dark/Light Mode
Add one toggle to change color scheme

### PR DESCRIPTION
**Pull request by boat46450**
Fixes #11

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Can you share a static deployment or a GIF?

**Comment by boat46450** [comment]
> Can you share a static deployment or a GIF?

Just like example ?

**Comment by aashutoshrathi** [comment]
No, exactly the changes.
Build it locally.

**Comment by boat46450** [comment]
https://gph.is/2NQlZv7

**Comment by aashutoshrathi** [comment]
Can you improve the toggle button? 
There are many beautiful buttons available for Day/Night mode switch.

**Comment by aashutoshrathi** [comment]
Refrences: https://codepen.io/jesperkc/pen/pydMqb

**Comment by SasiniSithara** [comment]
Refrences: https://codepen.io/jesperkc/pen/pydMqb

**Comment by boat46450** [comment]
Please recheck again
https://gph.is/2PbJ7kx

**Comment by aashutoshrathi** [comment]
@boat46450 the GIF is not clear but I think the moon icon is not in the right direction.

**Comment by Monal5031** [comment]
ping @boat46450

File: public/darkmode.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$('#toggleTheme').on('change', () => {
+  if ($('#toggleTheme').is(":checked")) {
+    $('body').removeClass('uk-section-secondary')
+    $('body').addClass('uk-section-muted')
+    console.log('>>> [darkmode.js:6] light ')
+  } else {
+    $('body').removeClass('uk-section-muted')
+    $('body').addClass('uk-section-secondary')
+    console.log('>>> [darkmode.js:10] dark ')
```
Comment (aashutoshrathi) [inline public/darkmode.js]: I think we don't need logging here anymore.

File: public/darkmode.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$('#toggleTheme').on('change', () => {
+  if ($('#toggleTheme').is(":checked")) {
+    $('body').removeClass('uk-section-secondary')
+    $('body').addClass('uk-section-muted')
+    console.log('>>> [darkmode.js:6] light ')
```
Comment (aashutoshrathi) [inline public/darkmode.js]: I think we don't need logging here anymore.

**Pull request closed without merging**